test/e2e/app_change_test.go

```diff
@@ -133,6 +133,8 @@ metadata:
 	assert.NoError(t, err)
 	secondDeploy, err := os.CreateTemp(os.TempDir(), "output2")
 	assert.NoError(t, err)
+	thirdDeploy, err := os.CreateTemp(os.TempDir(), "output3")
```

Comment:
I think this entire test should go into a separate file, discussed this with @praveenrewar and that could go in as a separate PR to keep this one less noisy

@praveenrewar good first issue maybe? October is around the corner 😆

Reply:
Yeah, separate PR makes sense 👍🏻